packs/data/extinction-curse-bestiary.db/cat-sith.json

```
@@ -487,7 +492,7 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p>@Template[type:emanation|distance:60]{60 feet} @UUID[Compendium.pf2e.bestiary-ability-glossary-srd.Aura]{Aura}</p>\n<p>Any creature within the area that can see the cat sith must succeed at a @Check[type:will|dc:21] save or become struck by an ill omen. On a failure, the creature becomes immune to fortune effects as long as it remains in the aura and for 1 minute thereafter. On a critical failure, the creature takes a -1 penalty to attack rolls, skill checks, and saving throws while within the ill omen aura and becomes immune to fortune effects for 1 day.</p>"
+                    "value": "<p>@Template[type:emanation|distance:60]{60 feet} @UUID[Compendium.pf2e.bestiary-ability-glossary-srd.Aura]{Aura}</p>\n<p>Any creature within the area that can see the cat sith must succeed at a @Check[type:will|dc:21] save or become struck by an ill omen. On a failure, the creature becomes immune to fortune effects as long as it remains in the aura and for 1 minute thereafter. On a critical failure, the creature takes a -1 penalty to attack rolls, skill checks, and saving throws while within the ill omen aura and becomes immune to fortune effects for 1 day.</p>\n<hr />\n<p>@UUID[Compendium.pf2e.bestiary-effects.Effect: Ill Omen]{Effect: Ill Omen}</p>"
```

```suggestion
                    "value": "<p>@Template[type:emanation|distance:60]{60 feet} @UUID[Compendium.pf2e.bestiary-ability-glossary-srd.Aura]{Aura}</p>\n<p>Any creature within the area that can see the cat sith must succeed at a @Check[type:will|dc:21] save or become struck by an ill omen. On a failure, the creature becomes immune to fortune effects as long as it remains in the aura and for 1 minute thereafter. On a critical failure, the creature takes a -1 penalty to attack rolls, skill checks, and saving throws while within the ill omen aura and becomes immune to fortune effects for 1 day.</p>\n<p>@UUID[Compendium.pf2e.bestiary-effects.Effect: Ill Omen]{Effect: Ill Omen}</p>"
```
Don't put a `<hr />` before the effect.

This suggestion may be broken unless github fixed their dumb bug.

---

Holy crap github finally fixed it! They closed my issue two months ago and told me the issue was going to take a while to fix.